src/Apps/Artist/Routes/Consign/Components/ArtistConsignSellArt.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { SectionContainer } from "./SectionContainer"
+
+export const ArtistConsignSellArt: React.FC = props => {
+  return (
+    <SectionContainer background="#EAE5E7">
```

Comment:
I guess you could use that `lightPurple` const that you made over in SectionContainer